<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MirrorAddDlg</class>
 <widget class="QWidget" name="MirrorAddDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>167</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="successLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelMirror">
       <property name="text">
        <string>Mirror:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelConnections">
       <property name="text">
        <string>Number of connections:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="url">
       <property name="clickMessage">
        <string>Enter an url</string>
       </property>
       <property name="showClearButton" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="numConnections">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPreference">
       <property name="text">
        <string>Preference:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KIntNumInput" name="preference">
       <property name="toolTip">
        <string>Optional: The preference of the mirror, 1 lowest 100 highest.</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="specialValueText">
        <string>not specified</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelLocation">
       <property name="text">
        <string>Location:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="KComboBox" name="location"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
